CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+# Cara Berkontribusi
+
+## Panduan Mulai Bekerja
+
+Dalam proyek ini setiap ada _issues_ dan mengerjakan _issue_ silahkan gunakan Github agar semua aktivitas dapat terdokumentasi dan dapat diikuti kemajuan pengerjaannya dari waktu ke waktu.
+
+Sebelum mengerjakan sebuah _issue_, ada beberapa hal yang harus diperhatikan:
+
+1. Silahkan isi kolom *assignee* dengan username masing-masing.
```

Comment:
I don't think this can be done by anybody other than the members of this repository.
So please remove it from this guide.